app/assets/javascripts/chat.coffee

```diff
@@ -0,0 +1,12 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
+submit_message = () ->
+  $('#message_body').on 'keydown', (event) ->
+    if event.keyCode is 13
+      $('input').click()
+      event.target.value = ""
+      event.preventDefault()
+
+scroll_bottom = () ->
```

Comment:
maybe you can try to move this function to helper.coffee or something like that, and use it anywhere